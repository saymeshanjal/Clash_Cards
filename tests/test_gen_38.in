100
14 3 2
8 46 3
8 59 1
8 175 4
16 182 1
6 116 4
10 122 1
14 75 2
4 121 3
2 125 4
10 19 2
8 159 5
14 162 2
16 5 2
10 148 4
10 46 2
12 106 4
6 64 4
8 137 2
16 90 4
4 54 1
14 49 5
6 159 4
4 174 4
16 34 2
10 127 5
10 140 1
16 78 5
10 18 4
8 97 1
14 154 5
12 144 5
8 66 1
8 179 3
2 59 4
4 39 1
2 187 1
4 26 2
6 79 5
12 170 4
4 24 3
10 166 4
2 1 3
8 197 2
12 157 3
6 79 4
12 159 4
12 57 5
8 35 4
8 32 4
10 42 1
16 166 4
6 54 5
2 121 4
6 39 2
14 97 1
6 22 2
4 155 1
6 22 2
8 97 1
12 125 3
2 60 4
4 103 5
10 15 2
8 33 3
2 64 2
10 50 1
2 179 2
10 77 1
10 176 4
12 94 4
16 110 2
2 40 5
14 84 2
14 95 1
12 128 2
12 60 2
10 108 1
14 25 3
14 149 5
2 114 1
2 114 1
10 162 1
16 16 2
4 151 3
12 100 1
4 170 3
16 30 4
2 36 1
16 114 5
14 11 2
8 51 3
14 47 2
6 54 5
4 87 1
2 133 4
16 54 4
8 54 1
8 6 4
10 45 3
