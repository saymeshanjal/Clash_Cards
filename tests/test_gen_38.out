330 1716 1386
27 43 0
35 35 0
8 42 20
6435 6435 0
1 14 5
126 126 0
1078 1079 1275
1 5 0
0 2 0
8 33 211
8 62 0
1078 1079 1275
495 4257 8118
44 133 75
76 77 99
208 441 275
1 14 5
20 21 29
3809 5406 3655
3 3 0
221 3211 0
1 14 5
0 5 1
254 1011 11605
45 207 0
126 126 0
3862 9008 0
4 222 26
35 35 0
987 2445 0
219 705 0
35 35 0
27 43 0
0 2 0
3 3 0
1 1 0
1 2 3
1 19 0
208 441 275
1 5 0
44 133 75
0 2 0
20 21 29
430 494 0
1 14 5
208 441 275
219 705 0
8 42 20
8 42 20
126 126 0
3809 5406 3655
1 19 0
0 2 0
5 6 9
1716 1716 0
5 6 9
3 3 0
5 6 9
35 35 0
430 494 0
0 2 0
0 6 0
7 41 204
27 43 0
0 1 1
126 126 0
0 1 1
126 126 0
44 133 75
208 441 275
4081 4082 4707
0 2 0
1078 1079 1275
1716 1716 0
286 287 351
286 287 351
126 126 0
577 2855 0
987 2445 0
1 1 0
1 1 0
126 126 0
499 2415 9956
1 5 0
462 462 0
1 5 0
350 10620 1900
1 1 0
4248 8622 0
135 729 2568
27 43 0
566 683 2183
1 19 0
3 3 0
0 2 0
523 7768 4579
35 35 0
0 66 4
110 142 0
